plugins/woocommerce/src/Internal/Admin/Marketing.php

```diff
@@ -140,6 +143,50 @@ protected function register_overview_page() {
 		}
 	}
 
+	/**
+	 * Order marketing menu items alphabeticaly.
+	 * Overview should be first.
```

Comment:
Add in info about Coupons page being the second item.

```suggestion
	 * Overview should be first, and Coupons should be second, followed by other marketing menu items.
```